specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/2016-12-01/backupManagement.json

```diff
@@ -1107,101 +1104,6 @@
         "deprecated": false
       }
     },
-    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/backupJobs": {
```

Comment:
They are moved to new file jobs.json with new API version as current one were buggy for Python SDK and Azure CLI.